scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -124,6 +138,7 @@ trait TypedDelimited[T] extends DelimitedScheme
   final override def sinkFields = fields
 }
 
+@deprecated("Use TypedText instead")
```

Comment:
I believe this one should be FixedTypedText? 
since 
FixedTypedText extends FixedPathSource with TypedTextDelimited
and 
FixedPathTypedDelimited extends FixedPathSource with TypedDelimited